### PR TITLE
Change get hello functionality to return a list of quotes.

### PR DESCRIPTION
**Pull request by cl2112**
Completed the json section (step 3) of the week 2 walkthrough.

### CODE REVIEW

File: portfolio/src/main/webapp/fetch.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/** Adds an onclick event listener to the get hello button. */
```
Comment (geoffbass) [inline portfolio/src/main/webapp/fetch.js:1]: Nice work on this script!

File: portfolio/src/main/webapp/fetch.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+/** Fetches a list of quotes from the server and adds one to the page. */ 
+async function showHello() {
+  const responseFromServer = await fetch('/hello');
+  const quotes = await responseFromServer.json();
+
+  // Pick a random quote.
+  const quote = quotes[Math.floor(Math.random() * quotes.length)];
+
+  const getHelloContainer = document.getElementById('get-hello-container');
+  getHelloContainer.innerHTML = quote;
```
Comment (geoffbass) [inline portfolio/src/main/webapp/fetch.js]: `innerHTML` can make your site vulnerable to XSS attacks and should be avoided (https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations). Use `innerText` or `textContent` instead.

**Review by geoffbass — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/fetch.js (hunk 2 above)
Comment (cl2112) [reply portfolio/src/main/webapp/fetch.js]: Wow, I did not know that, thank you for the information. I will fix it.

**Pull request merged**